test/test.js

```diff
@@ -851,3 +851,72 @@ test('with --quiet option does not print to stdout or stderr', async t => {
     t.is(error.exitCode, 1)
   }
 });
+
+test('--enable flag', async t => {
+  try {
+    await execa('../markdownlint.js',
+      ['--enable', 'MD002', '--config', 'default-false-config.yml', 'incorrect.md'],
+      {stripFinalNewline: false});
+    t.fail();
+  } catch (error) {
+    t.is(error.stdout, '');
+    t.is(error.stderr, 'incorrect.md:1 MD002/first-heading-h1/first-header-h1 First heading should be a top-level heading [Expected: h1; Actual: h2]\n');
+    t.is(error.exitCode, 1)
+  }
+});
+
+test('--enable flag does not modify already enabled rules', async t => {
+  try {
+    await execa('../markdownlint.js',
+      ['--enable', 'MD043', '--config', 'md043-config.yaml', 'correct.md'],
+      {stripFinalNewline: false});
+    t.fail();
+  } catch (error) {
+    t.is(error.stdout, '');
+    t.is(error.stderr, 'correct.md:1 MD043/required-headings/required-headers Required heading structure [Expected: # First; Actual: # header]\n');
+    t.is(error.exitCode, 1)
+  }
+});
+
+test('--enable flag accepts rule alias', async t => {
+  try {
+    await execa('../markdownlint.js',
+      ['--enable', 'first-header-h1', '--config', 'default-false-config.yml', 'incorrect.md'],
+      {stripFinalNewline: false});
+    t.fail();
+  } catch (error) {
+    t.is(error.stdout, '');
+    t.is(error.stderr, 'incorrect.md:1 MD002/first-heading-h1/first-header-h1 First heading should be a top-level heading [Expected: h1; Actual: h2]\n');
+    t.is(error.exitCode, 1)
+  }
+});
+
+test('--disable flag', async t => {
+  const result = await execa('../markdownlint.js',
+  ['--disable', 'MD002 MD014 MD022 MD041', 'incorrect.md'],
+  {stripFinalNewline: false});
+
+  t.is(result.stdout, '');
+  t.is(result.stderr, '');
+  t.is(result.exitCode, 0)
+
+  try {
+    await execa('../markdownlint.js',
+      ['--disable', 'MD014', 'incorrect.md'],
```

Comment:
I think the test failure is due to incorrect.md being treated as part of the disable list. See https://github.com/tj/commander.js/blob/master/docs/options-taking-varying-arguments.md#alternative-put-options-last for one way to avoid this, tho "--" is also fine and maybe good to show here.

Reply:
Yeah, I tried adding the `--` in there before but it didn't help. Or rather it wasn't enough. Turns out, you also need to pass each rules as a separate string for them to parse into an array, i.e.
```diff
- ['--disable', 'MD014 MD014 MD022', '--', 'incorrect.md']
+ ['--disable', 'MD014', 'MD014', 'MD022', '--', 'incorrect.md']
```
Do you prefer separating rules with spaces or commas? Spaces avoids the need for a custom parsing function but has slightly worse usability imo.